onnx/defs/tensor/defs.cc

```diff
@@ -682,7 +688,7 @@ ONNX_OPERATOR_SET_SCHEMA(
                   ")");
             }
           } else { // no value available for 'split'
-            int num_outputs = static_cast<int>(ctx.getNumOutputs());
+            int num_outputs = static_cast<int>(getAttribute(ctx, "num_outputs", 0));
```

Comment:
Would be good to check that the attribute is specified, and produce an error message otherwise. Similarly, it may be useful to check in the then-branch that the attribute is *not* specified.

Reply:
Done, please let me know if my changes address your comment.